zebra/if_netlink.c

```diff
@@ -1489,6 +1489,14 @@ int netlink_link_change(struct nlmsghdr *h, ns_id_t ns_id, int startup)
 		return 0;
 	}
 
+	if (ifi->ifi_change == IFF_PROMISC) {
```

Comment:
even if this identifies the changed flags accurately, what about other changes to other interface attributes? I don't see how we can bail out at this point in the processing.